Student/repository/student_repository.go

```diff
@@ -32,6 +32,8 @@ const (
 	WHERE id=$1;`
 	getSchoolName = `SELECT name FROM school WHERE ID=$1`
 	updateClasses = `UPDATE public.student SET current_classes=$1, classes_taken=$2, updated_at=$3 WHERE id = $4;`
+	search = `SELECT id, first_name, last_name, email, general_info, school, current_classes, classes_taken, created_at, updated_at
+	FROM public.student WHERE first_name LIKE $1 and last_name like $2 and current_classes @> $3`
```

Comment:
what happens if empty strings are passed as parameters, would all student results be returned? That might be a lot if we had a large amount of students, so if it does get to that point, we might want to limit the results returned. No need to change anything, this is just an idea I'm throwing out

Reply:
Yeah, if all parameters are empty it returns all the students. That's a good idea I'll add the limit for next sprint